clc/cmd/clc.go

```diff
@@ -255,11 +263,11 @@ func (m *Main) createCommands() error {
 	for _, c := range plug.Registry.Commands() {
 		c := c
 		// check if current command available in current mode
-		if !plug.Registry.IsAvailable(m.isInteractive, c.Name) {
+		if !plug.Registry.IsAvailable(m.mode != ModeNonInteractive, c.Name) {
 			continue
 		}
 		// skip interactive commands in interactive mode
-		if m.isInteractive {
+		if m.mode == ModeInteractive {
```

Comment:
With this users will be able to run shell in script mode right? Is this wanted?

Reply:
Scripts enhancements will be a separate PR.